### DISCUSSION TITLE
Raspbian OS arm64 Lite + manual install = problem with start of server

### DISCUSSION THREAD

**Discussion by bakotomas** [ravendb/ravendb#15242] [General]
Hello!

I need a help. I tried everything and now I'm stuck. I did this:

1.  downloaded  RavenDB-5.4.4-raspberry-pi.tar.bz2
2. extracted all to RavenDB folder in home directory
3. set chmod -R 755 ~/RavenDB
4. edited settings.json as:
```

{
    "ServerUrl": "http://192.168.0.10:9000",
    "Setup.Mode": "None",
    "DataDir": "/data/RavenData",
    "Security.UnsecuredAccessAllowed": "PrivateNetwork"
}
```

5. install dotnet : SDK 6.0.403, installed runtimes: 
   Microsoft.AspNetCore.App 5.0.17 [/home/guardian/.dotnet/shared/Microsoft.AspNetCore.App]
   Microsoft.AspNetCore.App 6.0.11 [/home/guardian/.dotnet/shared/Microsoft.AspNetCore.App]
   Microsoft.NETCore.App 5.0.17 [/home/guardian/.dotnet/shared/Microsoft.NETCore.App]
   Microsoft.NETCore.App 6.0.11 [/home/guardian/.dotnet/shared/Microsoft.NETCore.App]

6. tried to run server as daemon service with this result:  
`run.sh[149704]: /home/guardian/RavenDB/run.sh: line 11: /home/guardian/RavenDB/Server/Raven.Server: No such file or directory`

I'll appreciate any advice.

Tomas

**Comment by bakotomas** [comment]
OK, its probably arm32 build :-), I built server from source and its working now.

**Reply by garayx** [reply]
When I was getting this exception, switching to [ubuntu 20.04 arm64 deb package](https://hibernatingrhinos.com/builds?product=RavenDB&version=5.4&channel=Stable&pageNr=1) fixed it, I guess because it installs  necessary packages
